README.md

```diff
@@ -100,16 +98,9 @@ Below is an example of
 }
 ```
 
-To submit this request to Orchard:
-```html
-POST http://localhost:9001/v1/workflow
-```
-Which returns a workflow_id. For example: `wf-f231a08f-60e4-480a-b845-e53e06918f77`
+[sample_workflow.json.mustache](./example/data/sample_workflow.json.mustache) contains a mustache template which can accept these substitutions.
 
-Once defined, activate a workflow using the workflow id like so:
-```html
-PUT http://localhost:9001/v1/workflow/wf-f231a08f-60e4-480a-b845-e53e06918f77
-```
+Once generated, your `sample_workflow.json` can be used to create a workflow via `POST http://localhost:9001/v1/workflow`.
```

Comment:
Let's keep the activate part in the original README

`Once defined, activate a workflow using the workflow id like so:`
```html
PUT http://localhost:9001/v1/workflow/wf-f231a08f-60e4-480a-b845-e53e06918f77
```

Reply:
Moved the text around a bit to accommodate your suggestion